modules/lightning_features/lightning_media/src/Plugin/EmbedType/MediaLibrary.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\lightning_media\Plugin\EmbedType\MediaLibrary.
+ */
+
+namespace Drupal\lightning_media\Plugin\EmbedType;
+
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\StringTranslation\TranslationInterface;
+use Drupal\embed\EmbedType\EmbedTypeBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @EmbedType(
+ *   id = "media_library",
+ *   label = @Translation("Media Library")
+ * )
+ */
+class MediaLibrary extends EmbedTypeBase implements ContainerFactoryPluginInterface {
```

Comment:
I think this is just future-proofing on my part so that we have extra flexibility if we need it. But yes, at the moment we're not doing anything dramatically different from what Entity Embed already does.